### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Danjali**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		</div>
+
+		<div class="tab" id="tab2">
+			<a href="#tab2">Student Result</a>
+			<div class="content">
+			  Click on student name to check the details
+			</div>
+		</div>
+	</div>
+	<script src="dist/main.js"></script>
```
Comment (visha12l) [inline index.html]: remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+html,
```
Comment (visha12l) [inline css/style.css]: indent this file

File: src/index.js [hunk 3]
```diff
@@ -4,9 +4,9 @@ import { results, temp } from './named_export';
 let app = new App();
 
 
-app.getEvaluateResult(results);
-app.getPercentage(results);
-app.getTotalMarks(results);
-app.getListElement();
+// app.getEvaluateResult(results);
```
Comment (visha12l) [inline src/index.js]: remove commented code

File: src/index.js [hunk 4]
```diff
@@ -4,9 +4,9 @@ import { results, temp } from './named_export';
...
+// app.getPercentage(results);
+// app.getTotalMarks(results);
+app.evaluateResult(results);
 
 console.log(results);
```
Comment (visha12l) [inline src/index.js]: remove console line

File: src/function.js [hunk 5]
```diff
@@ -1,44 +1,69 @@
...
+			parentClass && list.classList.add(parentClass);
+			tabList.classList.add('active');
+			studentDetailsTab.textContent = "Student Details";
+			thirdTab.appendChild(tabList);
+			tabList.appendChild(studentDetailsTab);
+
+      //add properties to new tab
+			let secondTab = document.getElementsByTagName("li")[1];
+			secondTab.classList.remove('active');
+			var h1 = document.getElementsByTagName("a")[2];
```
Comment (visha12l) [inline src/function.js]: no use of var

File: src/function.js [hunk 6]
```diff
@@ -1,44 +1,69 @@
...
-          if(temp.english < 35 || temp.hindi < 35 || temp.mathematics <35){
-              item.result = 'fail';
-          }else{
-              item.result = 'pass';
-          }
-              results.map(item => item.result);
-        })
-      }
+	constructor() {
+		this.lists = results;
```
Comment (visha12l) [inline src/function.js]: what is the use of this.lists ?

File: src/function.js [hunk 7]
```diff
@@ -13,8 +12,7 @@ class App {
 				marks
 			} = item,
 			parentClass = (marks.english < 35 || marks.hindi < 35 || marks.mathematics < 35) ? 'redbg' : '',
-			percentage = (((marks.english + marks.hindi + marks.mathematics) / 300) * 100).toFixed(2),
-			totalMarks = marks.english + marks.hindi + marks.mathematics,
+			percentage = (((marks.english + marks.hindi + marks.mathematics) / 300) * 100).toFixed(2)+'%',
```
Comment (Danjali) [inline src/function.js:15]: use template literal string

File: src/function.js [hunk 8]
```diff
@@ -33,6 +31,7 @@ class App {
 			firstName.style.cursor = 'pointer';
 
 			firstName.onclick = function () {
+
```
Comment (Danjali) [inline src/function.js:34]: remove blank line

File: src/function.js [hunk 9]
```diff
@@ -59,11 +58,46 @@ class App {
 			h1.setAttributeNode(att2);
 
       //add content to third tab
+      results.filter(x => {
+        if(x.firstName === item.firstName){
+          let englishMarks = marks.english,
+              hindiMarks = marks.hindi,
```
Comment (Danjali) [inline src/function.js:64]: use object destructuring

File: src/function.js [hunk 10]
```diff
@@ -59,11 +58,46 @@ class App {
...
+  			     list1.classList.add('Heading');
+             name.textContent = item.firstName +' ' +item.lastName;
+  			     displayEnglish.textContent = englishMarks;
+  			     displayHindi.textContent = hindiMarks;
+  			     displayMathematics.textContent = mathematicsMarks;
+             displayTotalMarks.textContent = totalMarksStudent;
+             displayPecentage.textContent = pecentageStudent;
+
+             list1.appendChild(displayEnglish);
+  			     list1.appendChild(displayHindi);
```
Comment (Danjali) [inline src/function.js:87]: indentation

File: src/function.js [hunk 11]
```diff
@@ -59,11 +58,46 @@ class App {
...
+             list1.appendChild(name);
+             
+      }
+      });
+		};
+	});
+}
 	// createElement = tag => document.createElement(tag);
 
+ //evaluateResult();
```
Comment (Danjali) [inline src/function.js:101]: remove commented code

**Pull request merged**